templates/controller-metrics-reader-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.controller.enabled }}
```

Comment:
Do we want to configure the ClusterRole for controller metrics?

Reply:
Sorry, not sure what you're asking

Reply:
I dont think we need this file :)